.ci/publish-releasenotes-githubpage.sh

```diff
@@ -11,14 +11,5 @@ fi
 TARGET_VERSION=$1
 echo TARGET_VERSION="$TARGET_VERSION"
 
-CURRENT_VERSION=$(xmlstarlet sel -N pom=http://maven.apache.org/POM/4.0.0 \
```

Comment:
Can we remove xmlstartlet from installation in action ?

Reply:
Changes Done